app.js

```diff
@@ -1,14 +1,39 @@
-const handburger = document.querySelector(".handburger")
+const menuBtn = document.querySelector('.menu-btn');
+let menuOpen = false;
+menuBtn.addEventListener('click', () => {
+  if(!menuOpen) {
+    menuBtn.classList.add('open');
+    menuOpen = true;
+  } else {
+    menuBtn.classList.remove('open');
+    menuOpen = false;
+  }
```

Comment:
[OPTIONAL] Have you tried to use the toggle provided by Js? I know you haven't. You can try it, you will like it because, in a line of code, you will achieve all this. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle)